### PR TITLE
2 0 edit pages

### PR DESCRIPTION
**Pull request by ngilain**
Fix title on edition page
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2150
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/358

### CODE REVIEW

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Folder/FolderFormView.js [hunk 1]
```diff
@@ -36,6 +32,22 @@ class FolderFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
...
+     *
+     * @private
+     */
+    _renderForm() {
+        super._renderForm();
+        let title = $('#oo_folder_name', this.$el).val();
+        if (null === this._folderId) {
+            title = Translator.trans('open_orchestra_media_admin.table.folder.new');
+        }
+        $('#page-name', this.$el).html(title);
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Folder/FolderFormView.js]: why not pass value to a template

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaFormView.js [hunk 2]
```diff
@@ -41,6 +41,18 @@ class MediaFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
...
     }
+
+    /**
+     * Render a form
+     *
+     * @private
+     */
+    _renderForm() {
+        super._renderForm();
+        $('#page-name', this.$el).html($("input[id*='oo_media_']", this.$el).first().val());
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaFormView.js]: why not pass value to a template

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Folder/FolderFormView.js (hunk 1 above)
Comment (ngilain) [reply MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Folder/FolderFormView.js]: @alavieille : all my answers in https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2150

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaFormView.js (hunk 2 above)
Comment (ngilain) [reply MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaFormView.js]: @alavieille : all my answers in https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2150

**Comment by ngilain** [comment]
@alavieille : update done

**Review by alavieille — APPROVED** [review]

**Pull request merged**